queue_services/entity-filer/src/entity_filer/worker.py

```diff
@@ -96,23 +102,22 @@ def get_filing_types(legal_filings: dict):
     return filing_types
 
 
-async def publish_event(business: Business, filing: Filing):
+def publish_event(business: Business, filing: Filing):
```

Comment:
I think we have `entity-digital-credentials` listener is listening for nats queue messages.  Maybe `entity-bn` listener too.

Reply:
Are they listening on the generic queue?
I though the plan was for those to had dedicated topics, or is that a future?

Reply:
think there both listening on generic events queue.  

@vysakh-menon-aot think you are more familiar with entity-bn

Reply:
yes in entity-bn we are depending on it

Reply:
Thanks for checking Vysakh!

Thor - as per our conversation, let's add back the NATS message so both(GCP & NATS) messages are sent for each event.  Thanks!

Reply:
example https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/resources/v2/admin/administrative_bn.py#L44